config/app.default.php

```diff
@@ -9,7 +9,7 @@
      * Development Mode:
      * true: Errors and warnings shown.
      */
-    'debug' => true,
+    'debug' => env('DEBUG', true),
```

Comment:
`DEBUG` as an env var would be an int - i.e. 0 or 1

    'debug' => (boolean) env('DEBUG', true),

is therefore appropriate imo.